[enterprise-4.16] OSDOCS-10465: add RN entry (#76125)
openshift/openshift-docs: release_notes/ocp-4-16-release-notes.adoc

The Cluster Samples Operator is deprecated with the {product-title} 4.16 release. The Cluster Samples Operator will stop managing and providing support to the non-S2I samples (image streams and templates). No new templates, samples or non-Source-to-Image (Non-S2I) image streams will be added to the Cluster Samples Operator. However, the existing S2I builder image streams and templates will continue to receive updates until the Cluster Samples Operator is removed in a future release.

[id="ocp-4-16-rhel-worker-nodes-deprecation_{context}"]
==== Package-based {op-system-base} compute machines

> With this release, installation of package-based {op-system-base} worker nodes is deprecated. In a subsequent future release, {op-system-base} worker nodes is planned to be removed and no longer supported.

Reviewer: ```suggestion
With this release, installation of package-based {op-system-base} worker nodes is deprecated. In a subsequent future release, {op-system-base} worker nodes will be removed and no longer supported.
```
Does this change the meaning too much? I'm going for a more streamlined, minimal approach.

Author: i don't think it does 😄 thanks for the suggestion!

Reviewer 2: +1. Deprecations are one of the rare places where future tense is not only okay, but the "right thing to do" :tm:
